docs/r/cdap_streaming_program_run.md

```diff
@@ -5,13 +5,12 @@
 # Example
 
 ```
-resource "cdap_streaming_program_run" "example" {
-  namespace = "staging"
-  type      = "spark"
+resource "cdap_streaming_program_run" "test" {
+  namespace = "adp_staging"
   app       = "HL7v2_to_fhir"
 
   runtime_arguments = {
-    "system.profile.name" = "hl7-stream-ingest"
+    "system.profile.name" = "adp-aintegration-hl7-stream-ingest-us-central1-n"
```

Comment:
Should this value be replaced with a fake?

Reply:
sure.